primitives/src/transaction/transaction.rs

```diff
@@ -20,7 +20,6 @@ pub struct Transaction {
 impl Transaction {
     pub fn new(locktime: u32, inputs: Vec<Input>, outputs: Vec<Output>) -> Self {
```

Comment:
I think the locktime should be the final argument

Reply:
Unless you did it this way for memory layout reasons? Not a big deal either way

Reply:
I don't think there was a particular reason here to be honest, so I'll note to change this

Reply:
I don't think there was a particular reason here to be honest, so I'll note to change this